t/lib/t/Mechanize.pm

```diff
@@ -8,9 +8,7 @@ has mech => (
     is => 'ro',
     required => 1,
     lazy => 1,
-    default => sub {
-        MusicBrainz::WWW::Mechanize->new( catalyst_app => 'MusicBrainz::Server', quiet => 1 );
-    },
+    builder => 'make_mech',
```

Comment:
So that a test can run `make_mech` again to get another Mechanize instance (= browser session).